test/fork/airstream/CampaignLL.t.sol

```diff
@@ -55,10 +55,12 @@ abstract contract AirstreamCampaignLL_Fork_Test is Fork_Test {
                                           CREATE
         //////////////////////////////////////////////////////////////////////////*/
 
-        // Bound each leaf amount so that `campaignTotalAmount` does not overflow.
         Vars memory vars;
         vars.recipientsCount = params.leafData.length;
         for (uint256 i = 0; i < vars.recipientsCount; ++i) {
+            vm.assume(params.leafData[i].recipient != address(0));
```

Comment:
This will lead to many rejected tests. Instead, the recipient address should be bounded.

Reply:
True, how would it be the best way to do it? Is there an utility for bound address that I can't find on forge or should it be our own implemention?

Reply:
```solidity
    params.leafData[i].recipient = boundAddr(params.leafData[i].recipient);


    function boundAddr(address addr) internal pure returns (address) {
        return uint256ToAddr(_bound(addrToUint256(addr), 1, type(uint160).max - 1));
    }

    function addrToUint256(address x) internal pure returns (uint256) {
        return uint256(uint160(x));
    }

    function uint256ToAddr(uint256 x) internal pure returns (address) {
        require(x <= type(uint160).max, "overflow");
        return address(uint160(x));
    }
```

Reply:
> how would it be the best way to do it

Use a `uint256` seed instead of an address. See my [commit](https://github.com/sablier-labs/v2-periphery/pull/174/commits/f7b33523192a3eaf4e89414c167912b99527de4a).

Generally, the trick with fuzzing is to think of the function parameters as random numerical seeds, which can be later casted to any other type.